src/app/public/modules/page/index.ts

```diff
@@ -0,0 +1 @@
+export * from './page.module';
```

Comment:
Shouldn't we also be exporting `page-theme.service` here?

Also, do we want to add `export * from './page';` to the `/modules/index.ts` file?

Reply:
The page theme service is only an internal implementation, which is why I didn't add it to our export barrel.